packages/generator-electric/generators/app/templates/src/partials/OlderPosts.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+import Component from 'metal-component';
+import Soy from 'metal-soy';
+
+import templates from './OlderPosts.soy.js';
+
+class OlderPosts extends Component {
+	attached() {
+		this.filterPosts();
+	}
+
+	filterPosts() {
+		if (this.blogObject && this.url) {
+			const posts = [];
+
+			for (const i in this.blogObject.childIds) {
+				const childId = this.blogObject.childIds[i];
+				const post = this.blogObject.children[childId];
+
+				if (this.url !== post.url) {
+					posts.push(post);
+				}
+			}
+
+			const postsToShow = posts.lenth < 3 ? posts.lenth : 3;
```

Comment:
There something wrong here: `lenth` should be `length`

Reply:
Maybe we don't even need this line, apparently `slice` already do the job with you pass a number higher than you have on the array.